### PR TITLE
Feature/xformer2 ci build

### PR DESCRIPTION
**Pull request by lkindrat-xmos**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -17,9 +17,14 @@ RUN apt-get update && apt-get install -y \
     tcl environment-modules \
     && apt-get clean autoclean
 
-# install compiler
+# install gcc
```
Comment (ghost) [inline Dockerfile:20]: Are we using gcc to build the runtime?

File: Jenkinsfile [hunk 2]
```diff
@@ -49,11 +49,11 @@ pipeline {
...
                 // create venv and install pip packages
-                sh """conda env create -q -p ai_tools_venv -f ./utils/adf/environment.yml &&
-                      . activate ./ai_tools_venv &&
+                sh "conda env create -q -p ai_tools_venv -f ./utils/adf/environment.yml"
+                sh """. activate ./ai_tools_venv &&
                       pip install -e "./utils/adf/xcore_interpreters[test]" &&
-                      pip install -e "./tflite2xcore[test,examples,dev]" &&
-                      . deactivate"""
+                      pip install -e "./tflite2xcore[test,examples,dev]"
+                """
```
Comment (ghost) [inline Jenkinsfile:56]: Does the deactivate occur automatically? I am asking since it was removed.

File: Dockerfile (hunk 1 above)
Comment (lkindrat-xmos) [reply Dockerfile:20]: Yes, the runtime, the interpreters and a small component of xformer1 (`lib_flexbuffers`) are built with gcc. Do you think we should try building those with clang?

File: Jenkinsfile (hunk 2 above)
Comment (lkindrat-xmos) [reply Jenkinsfile:56]: The shell will close after the command is executed, so there is no need to deactivate. This is also why we need to run `. activate ...` in most steps before doing anything else. We could probably activate the conda env automatically, let's look at that another time.

Comment (ghost) [reply Jenkinsfile:56]: Alright

File: Dockerfile (hunk 1 above)
Comment (ghost) [reply Dockerfile:20]: There is no urgent need. We could move it to clang so that we use the same compiler everywhere.

**Review by ghost — APPROVED** [review]
+1

**Pull request merged**